version.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/ash
+#
+# shellcheck shell=dash
+#
+export SW_VERSION=0.0.10f
```

Comment:
ghost?

Reply:
Well ... no but after merging yes. It was still using discovery.sh which was good. Just removed it from discovery.